tests/test_ci/TestControllers.py

```diff
@@ -183,25 +183,53 @@ def test_customizedtest_added_to_queue(self):
         self.assertNotIn(1, customized_test)
 
     @mock.patch('mailer.Mailer')
-    def test_inform_mailing_list(self, mock_email):
+    @mock.patch('mod_ci.controllers.get_html_issue_body')
+    def test_inform_mailing_list(self, mock_get_html_issue_body, mock_email):
```

Comment:
This test doesn't seem to be fully right... The return value of the html_issue_body returns the same content passed to the send_simple_message, while there's a Jinja template parsing in between...

Reply:
Jinja template parsing is in html_issue_body which we have mocked here. So, there is no jinja involved in here and we are simply checking whether the function is called and it's return value is properly used or not.